lua/sniprun/display.lua

```diff
@@ -84,9 +92,12 @@ function M.write_to_term(message, ok)
     vim.api.nvim_chan_send(M.term.chan, line)
     vim.api.nvim_chan_send(M.term.chan, "\n\r");
   end
-  vim.api.nvim_chan_send(M.term.chan, "\n\r");
+
   M.term.current_line = h
 
+  if M.term.current_line > vim.fn.line("w$") then
```

Comment:
could you explain the purpose (and effect) of those lines ?

Reply:
@michaelb These are for scrolling to last output line (M.term.current_line). If last line > window visible line (line("w$")), which means last line is invisible, jump to it. The use case here is user can scroll output window with long scrollback lines, and jump to the latest line after running new code.

Reply:
@michaelb Previously `set scrollback=1` keeps only visible content so didn't need those lines.